examples/.config/model_params_pytorch.json

```diff
@@ -706,37 +697,22 @@
       "model_src_dir": "object_detection/ssd_resnet34/quantization/ptq/fx",
       "dataset_location": "/tf_dataset2/datasets/coco2017/resized_1200x1200/coconpy",
       "input_model": "/tf_dataset2/models/pytorch/ssd_resnet34/resnet34-ssd1200.pytorch",
-      "yaml": "conf.yaml",
-      "strategy": "basic",
-      "batch_size": 1,
-      "new_benchmark": false
+      "main_script": "python/main.py",
+      "batch_size": 1
```

Comment:
Can we use bigger bs, so run accuracy will be faster.

Reply:
If we change here to bigger bs, where should we set the batch_size=1 for performance? It is unclear to me